app/services/kredits.js

```diff
@@ -115,6 +118,14 @@ export default Service.extend({
     return contributor;
   },
 
+  buildProposal(attributes) {
+    debug('[kredits] buildProposal', attributes);
+
+    let proposal = getOwner(this).lookup('model:proposal');
+    proposal.setProperties(attributes);
+    return proposal;
```

Comment:
This looks a bit strange to me. Is there a benefit to not just importing the model and doing a `create()` or something? (I'm not actually sure what's really happening here though.)

Reply:
The problem with the `.create()` is that we can not inject services because the model doesn't have a container. I'm unhappy with this as well and will try to figure out a better way.